packages/discovery-react-components/src/components/SearchResults/utils/__tests__/getDocumentTitle.test.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { getDocumentTitle } from '../index';
+
+const mockResultWithTitle = {
+  document_id: 'df4c085040a9049ddf956ca6a213699d',
+  extracted_metadata: {
+    file_type: 'html',
+    title: 'Family Day 2020 (Israel)'
+  }
+};
+
+const mockResultWithoutTitle = {
+  document_id: 'df4c085040a9049ddf956ca6a213699d',
+  extracted_metadata: {
+    file_type: 'html'
+  }
+};
+
+const mockResultWithFilename = {
+  document_id: 'df4c085040a9049ddf956ca6a213699d',
+  extracted_metadata: {
+    file_type: 'html',
+    filename: 'Family Day 2020 (Israel)'
+  }
+};
+
+describe('getDocumentTitle', () => {
+  test('returns result title', () => {
```

Comment:
Is there a reason you are using `test` instead of `it`?

Reply:
to be consistent with other test in component